pkg/utils/version.go

```diff
@@ -0,0 +1,29 @@
+package utils
+
+import (
+	"context"
+
+	semverlib "github.com/Masterminds/semver/v3"
+	"github.com/google/go-github/v45/github"
+)
+
+func CheckLatestVersion(currentVersion string) {
+	client := github.NewClient(nil)
+	latestRelease, _, err := client.Repositories.GetLatestRelease(context.Background(), "pluralsh", "plural-cli")
```

Comment:
This will returned latest release looking at date not version, see https://docs.github.com/en/rest/releases/releases#get-the-latest-release.

Reply:
If we would create any patch release then it would not work.

Reply:
I also get tag release and it returns the version, exactly what you did

Reply:
We'll need to order it by version not date to fix that.

Reply:
@maciaszczykm you are right. I will fix it. Thanks